lib/u3d/commands.rb

```diff
@@ -107,6 +107,10 @@ def download(args: [], options: {})
         cache = Cache.new(force_os: os)
         versions = cache[os.id2name]['versions']
         version = interpret_latest(version, versions)
+        unless versions[version]
+          UI.error "No version #{version} was found in cache"
```

Comment:
`Version #{version} was not found. Either it doesn't exist or u3d doesn't know about it yet. Try refreshing with u3d available -f`